content/exporters/custom-exporter/Node.js/Metrics.md

```diff
@@ -73,25 +73,34 @@ export class MyConsoleStatsExporter implements StatsEventListener {
 And now to test it out as we would in a typically linked program, let's create a `expample.js` file:
 
 {{<highlight javascript>}}
-var opencensus = require('@opencensus/core');
-var stats = new opencensus.Stats();
+const { globalStats, AggregationType, TagMap } = require('@opencensus/core');
 
 // Let's create an instance of our just created exporter
-var exporter = new MyConsoleStatsExporter();
+const exporter = new MyConsoleStatsExporter();
 // And register it
-stats.registerExporter(exporter);
+globalStats.registerExporter(exporter);
```

Comment:
Why are we changing this to `globalStats`? What's up with `stats`? Other languages use the same or is another `stats` used somewhere?

Reply:
Earlier we used to create Stats instance (```new Stats()```) and used in the application. With new release, we have changed it to a global singleton Stats instance (```globalStats```). This behavioral change is introduced to match other language implementation. (The name ```globalStats``` is specific to Node library).